src/platforms/posix/drivers/df_hmc5883_wrapper/df_hmc5883_wrapper.cpp

```diff
@@ -75,7 +75,7 @@ using namespace DriverFramework;
 class DfHmc9250Wrapper : public HMC5883
 {
 public:
-	DfHmc9250Wrapper(enum Rotation rotation);
+	DfHmc9250Wrapper(enum Rotation rotation, const char *path);
```

Comment:
Hmc9250?

Reply:
For some reason it's always been numbered like that (at least since I've worked with PX4), and I've worked with it enough that I just kinda gloss over the error anymore. I can fix it real quick.

Reply:
I'm guessing it was copied from the mpu9250 and not properly renamed.